vars/fcosKola.groovy

```diff
@@ -14,6 +14,7 @@ def call(params = [:]) {
     // this is shared between `kola run` and `kola run-upgrade`
     def platformArgs = params.get('platformArgs', "");
     def buildID = params.get('build', "latest");
+    def arch = params.get('arch', "x86_64");
```

Comment:
Hmm instead of hardcoding x86_64, why not just leave it empty and not pass any `arch` arg at all. E.g.

```suggestion
    def arch = params.get('arch', "");
    if (arch != "") {
        arch = "--arch=${arch}"
    }
```

That way `kola` should default to the arch we're running on, which seems more proper. But OK as is too, we won't have any Jenkins using this code on other arches anytime soon.